Sources/Preferences/Preferences.swift

```diff
@@ -79,6 +79,14 @@ extension Preferences {
     public func reset() {
       value = defaultValue
     }
+    
+    // An accessor to `value` which is MainActor isolated in case we need to ensure
+    // Note it would be best to make `value` MainActor isolated
+    // but this results in very large changes
+    @MainActor public var isolatedValue: ValueType {
+      get { return value }
+      set { value = newValue }
+    }
```

Comment:
This doesn't make sense to add, either a type _requires_ actor isolation for a specific reason or it doesn't, you shouldn't add a new property thats isolated that sits side-by-side with a nonisolated version.

Need to reconsider _why_ you need actor isolation for a pref and fix that instead. If the whole thing is that all prefs need to be main actor isolated in general, then it should annotated on the class itself, not on a new property

Reply:
Ok I'll remove this, this isn't the fix it's just some compiler help. But I suggest we make an issue to isolate Preferences.

Reply:
@kylehickinson should we main actor isolate preferences in general?
Then I would even make it in separate commit